app/src/main/java/com/eveningoutpost/dexdrip/Services/WixelReader.java

```diff
@@ -294,117 +285,114 @@ public static boolean almostEquals( TransmitterRawData e1, TransmitterRawData e2
+                        lastReportedReading = lastReading;
+                        lastReportedTime = lastReading.CaptureDateTime;
+                    }
+                }
+                // let's sleep (right now for 30 seconds)
+                Thread.sleep(30000);
+            }
+        } catch (InterruptedException e) {
+            Log.e(TAG, "cought InterruptedException! ", e);
             // time to get out...
         }
     }
 
     // this function is only a test function. It is used to set many points fast in order to allow
     // faster testing without real data.
-    public void runFake()
-    {
+    public void runFake() {
         // let's start by faking numbers....
         int i = 0;
         int added = 5;
         while (!mStop) {
             try {
 
-                i+=added;
-                if (i==50) {
+                i += added;
+                if (i == 50) {
                     added = -5;
                 }
-                if (i==0) {
+                if (i == 0) {
                     added = 5;
                 }
 
                 int fakedRaw = 100000 + i * 3000;
                 Log.e(TAG, "calling setSerialDataToTransmitterRawData " + fakedRaw);
-                setSerialDataToTransmitterRawData(fakedRaw, fakedRaw ,100, new Date().getTime());
+                setSerialDataToTransmitterRawData(fakedRaw, fakedRaw, 100, new Date().getTime());
                 Log.e(TAG, "returned from setSerialDataToTransmitterRawData " + fakedRaw);
 
                 Long StartLoop = new Date().getTime();
-                for (int j = 0 ; j < 300; j++) {
+                for (int j = 0; j < 300; j++) {
                     Thread.sleep(1000);
-                    Log.e(TAG, "looping ...." + i + " " + j + " " + (new Date().getTime() - StartLoop)/1000);
-                    if(mStop ) {
-                    // we were asked to leave, so do it....
-						Log.e(TAG, "EXITING mstop=true" );
+                    Log.e(TAG, "looping ...." + i + " " + j + " " + (new Date().getTime() - StartLoop) / 1000);
+                    if (mStop) {
+                        // we were asked to leave, so do it....
+                        Log.e(TAG, "EXITING mstop=true");
                         return;
                     }
                 }
 
 
-               } catch (InterruptedException e) {
-                   // time to get out...
-                   Log.e(TAG, "cought InterruptedException! ", e);
-                   break;
-               }
+            } catch (InterruptedException e) {
+                // time to get out...
+                Log.e(TAG, "cought InterruptedException! ", e);
+                break;
+            }
         }
-		Log.e(TAG, "EXITING mstop=true" );
+        Log.e(TAG, "EXITING mstop=true");
     }
 
-    public void Stop()
-    {
+    public void Stop() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1313443989/issues/source?bid=2735053&fileBranchId=2735067#l390)